docgen/src/stylesheets/components/_sidebar.scss

```diff
@@ -7,10 +7,12 @@ nav.sidebar {
   border-right: 1px solid #d8d8d8;
   overflow: hidden;
   padding-top: 120px;
+  user-select: none;
 
   .sidebar-container {
     width: 100%;
     max-width: $sidebar-width;
+    user-select: all;
```

Comment:
It's used to re-set back the `user-select: none` of above. 
<details><img src="https://puu.sh/sWDwQ/8f85ebe9e1.png"></details>